DEMOQA/cypress/tests/login.js

```diff
@@ -0,0 +1,204 @@
+describe('login', () => {
+
+    const testData = require('../fixtures/testData.json')[0]
+    beforeEach(() => {
+        cy.visit("https://demoqa.com/automation-practice-form")
+    });
+    it('submit successfully with valid input', () => {
+        cy.fixture('testData').then((data) => {
```

Comment:
This is not wrong, but I prefer `cy.fixture("file").as("var")` in `beforeEach()` and then `cy.get("@var").then()` here